contents/blog/posthog-marketing.md

```diff
@@ -0,0 +1,211 @@
+---
+date: 2022-09-23
+title: "How our marketing team uses PostHog"
```

Comment:
Possible alternative titles:

1. How (and why) you should use PostHog on your website, too
2. How we Posthog for marketing (and why you should too)

I like (2), but I think I need to re-write / add a stronger conclusion to support that headline. Something that bascially sums up the benefits.

Reply:
Actually settling on 'How (and why) our marketing team uses PostHog' atm the moment